include/CSVManager.h

```diff
@@ -0,0 +1,12 @@
+#include<vector>
+#include<string>
+
+class CSVManager {
+    std::vector<std::vector<std::string>> data;
+public:
+    std::vector<std::string> readLine(std::string);
+    void readFile(std::string);
+    void writeFile(std::string);
```

Comment:
I think the `writeFile` api here has no use because we usually don't edit a csv file.